static/app/views/ddm/sampleTable.tsx

```diff
@@ -95,9 +106,14 @@ export function SampleTable({mri, ...metricMetaOptions}: SamplesTableProps) {
             <StyledPlatformIcon platform={project?.platform || 'default'} />
           </Tooltip>
           <Link
-            to={getTransactionDetailsUrl(organization.slug, eventSlug, undefined, {
-              referrer: 'metrics',
-            })}
+            to={normalizeUrl(
+              `/organizations/${organization.slug}/performance/summary/?${qs.stringify({
+                ...location.query,
```

Comment:
Would this carry over all the URL parameters from DDM to the transaction summary? I don't think we want this behavior.

We only care about page filters, so we could likely use
https://github.com/getsentry/sentry/blob/56f57d6c7ad9e396e5f07f3a32d126a77a4f160f/static/app/components/globalSelectionLink.tsx#L35
or 
https://github.com/getsentry/sentry/blob/6d59d8e02b72983584fc90d19178bb029a27c3d3/static/app/components/organizations/pageFilters/utils.tsx#L34
instead.

Reply:
Thanks! Fixed in [8ccb7a2](https://github.com/getsentry/sentry/pull/63239/commits/8ccb7a2d461a736745e398cce10a2b6c9031ed53)